chia/util/keyring_wrapper.py

```diff
@@ -187,17 +202,29 @@ def set_keys_root_path(keys_root_path: Path):
     @staticmethod
     def get_shared_instance(create_if_necessary=True):
         if not KeyringWrapper.__shared_instance and create_if_necessary:
-            KeyringWrapper.__shared_instance = KeyringWrapper(keys_root_path=KeyringWrapper.__keys_root_path)
+            KeyringWrapper.__shared_instance = KeyringWrapper.create(keys_root_path=KeyringWrapper.__keys_root_path)
 
         return KeyringWrapper.__shared_instance
 
+    @staticmethod
+    def set_shared_instance(instance: "KeyringWrapper"):
```

Comment:
I don't see any calls to this function. It seems like a risky hole in the abstraction too. Can it be removed?

Reply:
Oh, if this is added by my patch, this is probably leftover code - I'll remove it.

Reply:
removed